static/index.html

```diff
@@ -38,8 +38,8 @@
                             <th width="150">Assignee</th>
                             <th width="150">Milestone</th>
                             <th width="100">State</th>
+                            <th width="100">Age (in hours)</th>
                             <th>Repo</th>
-                            <th>HTML</th>
```

Comment:
ok please can you come back and advise me on this. Because we dont need to show the link. Only use it to make the table more usable.